content/ja/case-studies/spotify/index.html

```diff
@@ -0,0 +1,120 @@
+    
+<br><br>
+
+チームは、本マイグレーションに必要なコアテクノロジーの問題に対応するため、チームは2018年の大半を費やしました。「レガシーのインフラをサポートしたり連携するKubernetes APIやKubernetesの拡張性機能をたくさん使うことができたので、インテグレーションはシンプルで簡単なものでした」とサイト・リライアビリティ・エンジニアのJames Wenは言います。
+
+<br><br>
+マイグレーションはその年の後半に始まり、2019年に加速しました。「私たちはステートレスなサービスに注力しています。最後に残るテクノロジーブロッカーを突破したら、それが上昇をもたらしてくれると期待しています」とChakrabartiは言います。「ステートフルサービスについては、より多くのやるべきことがあります。」
+<br><br>
+今のところ、Spotifyの150を超えるサービスのごく一部がKubernetesに移行されています。
+
+「社内のチームから聞こえてきたのは、マニュアルでのキャパシティプロビジョニングを意識する必要性が少なくなり、Spotifyとしての機能の提供に集中できる時間がより多くなってきたということです」とChakrabartiは言います。
+
+Kubernetesで現在実行されている最も大きなサービスはアグリゲーションサービスで、1秒あたり約1000万リクエストを受け取り、オートスケールによる大きな恩恵を受けている、とWenは言います。さらに、「以前はチームが新しいサービスを作り、運用ホストを本番環境で稼働させるために1時間待たなければなりませんでしたが、Kubernetesでは秒・分のオーダーでそれを実現できます」と付け加えます。さらに、Kubernetesのビンパッキング（組み合わせ最適化）機能やマルチテナント機能により、CPU使用率が平均して2〜3倍向上しました。
+
+
+</div>
+</section>
+<div class="banner4" style="background-image: url('/images/CaseStudy_spotify_banner4.jpg')">
+  <div class="banner4text">
+  「レガシーのインフラをサポートしたり連携するKubernetes APIやKubernetesの拡張性機能をたくさん使うことができたので、インテグレーションはシンプルで簡単なものでした」<br style="height:25px"><span style="font-size:14px;letter-spacing:2px;text-transform:uppercase;margin-top:5% !important;"><br>-  Spotify、Spotifyエンジニア、James Wen</span>
+  </div>
+</div>
+
+<section class="section5" style="padding:0px !important">
+<div class="fullcol">
+  Chakrabartiは、Spotifyが見ている4つのトップレベルのメトリック - リードタイム、デプロイ頻度、修復時間、そして運用負荷 - のすべてについて「Kubernetesがインパクトを与えている」と指摘します。
+<br><br>
+Kubernetesが初期の頃に出てきたサクセスストーリーの1つに、SpotifyチームがKubernetesの上に構築したSlingshotというツールがあります。「プルリクエストを出すと、24時間後に自己消滅する一時的なステージング環境を生成します」とChakrabartiは言います。「これはすべてKubernetesがやってくれています。新しいテクノロジーが出てきて使えるようになったときに、自分のイメージを超えるようなソリューションをいかにしてこの環境上で作っていくか、そのやり方を示す刺激的な例だと思います。」
+<br><br>
+またSpotifyは<a href="https://grpc.io/">gRPC</a>と<a href="https://www.envoyproxy.io/">envoy</a>を使い、Kubernetesと同じように、既存の自社製ソリューションを置き換え始めました。「私たちはその時の自分たちの規模を理由にした開発をしていて、実際に他のソリューションはありませんでした」とインフラおよび運用担当のソフトウェアエンジニアであるDave Zolotuskyは言います。「しかし、そういった規模感のツールですらコミュニティは私たちに追いつき、追い越して行きました。」
+
+
+</div>
+
+<div class="banner5" >
+  <div class="banner5text">
+    「私たちが取り組んでいることに関する専門知識を得るために、コンタクトしたい人と連絡を取るのは驚くほど簡単でした。そして、私たちがしていることのすべてを検証することを助けてくれました。」 <br style="height:25px"><span style="font-size:14px;letter-spacing:2px;text-transform:uppercase;margin-top:5% !important;"><br>-  Spotify、サイト・リライアビリティ・エンジニア、James Wen</span></div>
+</div>
+
+
+<div class="fullcol">
+  どちらのテクノロジーも採用するには初期段階ではありますが、「gRPCはスキーマ管理、API設計、下位互換の問題など、初期の開発段階における多くの問題をより劇的な影響を与えると確信しています」とZolotuskyは言います。「なのでそういった領域でgRPCに傾倒しています。」
+
+<br><br>
+チームはSpotifyのクラウドネイティブなスタックを拡大し続けており - この次にあるのはスタック トレーシングです -  CNCFランドスケープを有用なガイドとして活用しています。「解決する必要があるものを見たときに、もし多数のプロジェクトがあればそれらを同じように評価しますが、そのプロジェクトがCNCFプロジェクトであることには間違いなく価値があります」とZolotuskyは言います。
+
+<br><br>
+SpotifyがKubernetesでこれまでに経験してきたことはこれを裏付けています。「このコミュニティは、あらゆるテクノロジーにより速くより簡単に取り組めるようになる点で極めて有益です」とZolotuskyは言います。「私たちが取り組んでいることに関する専門知識を得るために、コンタクトしたい人と連絡を取るのは驚くほど簡単でした。そして、私たちがしていることのすべてを検証することを助けてくれました。」
```

Comment:
- これ、これ、と連続している
- により（に起因して）と読みとれてしまいそうだった部分を修正

```suggestion
SpotifyがKubernetesでこれまでに経験してきたことはそれを裏付けています。「このコミュニティは、あらゆるテクノロジーをより速くより簡単に取り組めるようになる点で極めて有益です」とZolotuskyは言います。「私たちが取り組んでいることに関する専門知識を得るために、コンタクトしたい人と連絡を取るのは驚くほど簡単でした。そして、私たちがしていることのすべてを検証することを助けてくれました。」
```